pkg/store/promproxy_test.go

```diff
@@ -112,3 +112,62 @@ func TestPrometheusProxy_LabelValues(t *testing.T) {
 
 	testutil.Equals(t, []string{"a", "b", "c"}, resp.Values)
 }
+
+func TestPrometheusProxy_Series_MatchExternalLabel(t *testing.T) {
+	p, err := testutil.NewPrometheus(":12347")
```

Comment:
We should probably switch this to grab a random free port automatically and add a getter like `p.Addr()`. Bumping the port number by 1 for every test is probably not sustainable :D